WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift

```diff
@@ -167,6 +176,18 @@ private extension TopBannerView {
         return iconInformationStackView
     }
 
+    func createLabelHolderStackView() -> UIStackView {
+        labelHolderStackView.addArrangedSubviews([
+            createSeparatorView(height: 48, width: 0),
```

Comment:
Could we add these values into constants? That way we can reuse them, and we know what they stand for

Reply:
Definitely! https://github.com/woocommerce/woocommerce-ios/pull/8728/commits/4579e44d607237413e6b1fb4dd4e5a5c87a70e10